### ISSUE TITLE
[Bug]: Pokemon Crystal Weird Visual Stutter when starting Battle

### ISSUE THREAD

**Issue by Myfharad** [visualboyadvance-m/visualboyadvance-m#973]
### Expected behavior?

I assume a good majority of people have seen how pokemon battles in pokemon crystal start

Your Character model and the opposing pokemon sprite sideswipe onto the screen to fight like normal.

### Actual behavior?

When I get into any battle the game kinda just stutters a bit, not in a FPS drop kind of way (atleast for what I think)

Unlisted Video of what I see: https://youtu.be/LJqXAE7HN10

### Steps to reproduce the problem

If its on my side it may be something I have enabled or disabled, but I wanted to check with you all incase it was an active bug

### Error messages?

If this is a crash bug, and you have the time and inclination, please provide a symbolic backtrace.
Instructions for doing so are [here](https://github.com/visualboyadvance-m/visualboyadvance-m/blob/master/README.md#reporting-crash-bugs).

### What game were you playing

<Some of this information can be provided with a screenshot of Rom Information, SAVE TYPE for GBA titles can be obtained from Options > Game Boy Advance > Config>
Title: Pokemon Crystal
GB, GBC or GBA: GBC
ROM CODE: 
BIOS Used?: (Share crc hash)
SAVE TYPE: (Attach Battery or Savestate here)
Clean, Intro'd or Hacked?:(Attach Patch here)

### What settings are you using?

Video driver (simple, opengl): OpenGL
Sound driver (directsound, xaudio, openal, SDL): DirectSound (I also tried every other sound driver)
Scale Filter Used?: Simple2x 

### Specifications and Build details

VBA-M: (include commit version number and whether you are using the 32-bit or 64-bit version)  2.1.4 64bit
OS: (include version and OS build number using WinVer if on Windows): ??? Windows 10
If using Linux, specify if using xorg or Wayland:
CPU: i7-7700k
GPU: EVGA GTX 1070
RAM: 16gb 

### Copyright Compliance

- [X] There are no infringing files attached to this bug report.

**Comment by ZachBacon** [comment]
So first of all, I'm going to ask that you try one of the nightlies from https://nightly.vba-m.com/ try that and then report back.

**Comment by Myfharad** [comment]
> 

Just tried it, having the same visual bug

**Comment by Myfharad** [comment]
Would you happen to know if its just a me problem? or has anyone else who played Crystal recently seen the visual bug Im getting?

**Comment by Squall-Leonhart** [comment]
there is a delay, but that delay is also there for moving between area's (ie, into and out of caves) and opening or closing the menus and map.

i want to check on the vc version of crystal.

**Comment by Myfharad** [comment]
I've noticed the delay, but its present on other GBA emulators, so Im chalking it up to the game itself.

But from the YouTube video I posted on the Issue itself, I want to know if anyone else is experiencing the weird VISUAL STUTTER (https://youtu.be/LJqXAE7HN10) when the trainer's sprite and the opposing trainer/pokemon's sprite are placed ont he field.

**Comment by Squall-Leonhart** [comment]
no, thats not supposed to happen, the window scrolling is off for some reason.

**Comment by Squall-Leonhart** [comment]
@endrift this issue happens under gold and silver in DMG mode which is probably a limitation of the dmg LCD capabilities, is it INT 48 timing related?

**Comment by EternityEngine** [comment]
Hi, I noticed the battle intro scroll jitter happens sometimes when I play a Crystal hack (Pokemon Fool's Gold) at 2x size in both 2.1.11 and nightly 2.1.11-9d20ce9b (64- and 32-bit), but not with my old MFC version (2.0.0 (SVNGit:), Sep 28 2015, 32-bit).  The jitter might be somewhat timing-based at sizes where the pixel shifting is not apparent; I'm finding it quite hard to replicate. However, there is noticeable horizontal pixel shifting at 3x size, visible both in the battle intro scroll and when moving sideways in a location with a large amount of tiled graphics (Goldenrod City, for example). It's somewhat visible at 5x and 6x size as well, but not visible at 4x, though I got a noticeable jitter once at 4x as well. I also got some jitters at 1x, which seems to indicate that it is not an issue with the size magnification specifically.

The difficulty of replicating the jitter makes it hard to check for certain whether it occurs in the MFC version (though I ran enough battles to be pretty sure it doesn't), but regardless of that, the obvious horizontal pixel shifting at 3x definitely doesn't occur.

The delay for moving between areas and opening the menu happens consistently on all versions I've mentioned, so that might be built-in.

**Comment by Squall-Leonhart** [comment]
can only replicate this with GB games, GBA games scale from 1x to 2x and back and back to 2x fine
